### PR TITLE
Countdown

### PR DESCRIPTION
**Pull request by London804**
I feel like this can be done better I just don't know how.

### CODE REVIEW

File: countdown.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
 function countdown(seconds){
-  // ...
+	var tick = 0;
+
+	setInterval(function() {
+        if (seconds > tick) {
+        	seconds--;
+        	console.log(seconds);
+        }
```
Comment (afeld) [inline countdown.js:8]: What happens when `seconds` reaches zero? Does the interval stop?

**Comment by afeld** [comment]
Just one note – looks good otherwise!

File: countdown.js (hunk 1 above)
Comment (London804) [inline countdown.js:8]: Yes.

Comment (afeld) [inline countdown.js:8]: Well, it stops printing numbers, but the interval continues to run. Can you make it stop once it's finished the countdown?

**Pull request closed without merging**